### DISCUSSION TITLE
Wapplyzer detects Fingerprintjs!

### DISCUSSION THREAD

**Discussion by mohaa7** [fingerprintjs/fingerprintjs#836] [Q&A] [answered]
[Wappalyzer](https://www.wappalyzer.com/apps) is a browser extension that detects the use of opensource fingerprintjs, and users who have this addon enabled can easily determine whether or not your site is tracking by fingerprintjs.
Is there a way to hide fingerprints from the Wapplyzer?

**Comment by makma** [comment, marked as answer]
Hello @mohaa7,
Since JavaScript is an interpreted language and it's executed on the client side, any extension/user can observe downloaded or executed code. The library's responsibility is to fingerprint, not to hide its content. If the detection is an issue for you, I'd recommend obfuscating your web application.

**Comment by mohaa7** [comment]
It seems no longer recognized by the Wapplyzer!
